include/moves.h

```diff
@@ -77,8 +77,8 @@ void UnSetMove(s32 index, Move *moves);
 
 u8 sub_809287C(Move *move);
 void sub_80928A0(u8 *buffer, Move *move, const unkStruct_80928C0 *);
-void sub_80928C0(u8 *buffer, Move *move, const unkStruct_80928C0 *);
-void sub_8092AA8(Move *move, u16 moveID);
+void GetMoveName(u8 *buffer, Move *move, const unkStruct_80928C0 *);
```

Comment:
Shouldn't this be BufferMoveName similar to BufferItemName? Just want to be consistent.

Reply:
Yeah, I'll rename this.